tutorial/creating_packages/preparing_the_build.rst

```diff
@@ -93,7 +94,8 @@ As you can see:
 * Based on the value of the ``with_fmt`` option:
 
     - We install or not the ``fmt/8.1.1`` Conan package.
-    - We require or not a minimum C++ standard as the *fmt* library requires at least C++11.
+    - We require or not a minimum and a maximum C++ standard as the *fmt* library requires at least C++11 and our
+      settings support at most C++14.
```

Comment:
```suggestion
    - We require or not a minimum and a maximum C++ standard as the *fmt* library requires at least C++11 and it will not compile if we try to use a standard above C++14 (just an example, *fmt* can build with more modern standards)
```